gtk/tests_backend/screen.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+import pytest
+from gi.repository import GdkX11
+
+from .probe import BaseProbe
+
+
+class ScreenProbe(BaseProbe):
+    def __init__(self, screen):
+        super().__init__()
+        self.screen = screen
+        self._impl = screen._impl
+        self.native = screen._impl.native
+        if os.environ.get("XDG_SESSION_TYPE", "").lower() == "x11":
+            assert isinstance(self.native, GdkX11.X11Monitor)
+        else:
+            # TODO: Check for the other monitor native types
```

Comment:
Is this TODO still active? I would have thought we'd know what the answer is on Wayland?

Reply:
I know the answer, it's just that I cannot directly import `__gi__.GdkWaylandMonitor` for type checking.